### PR TITLE
[MB-15483] Create a new order type for "Local Move"

### PR DESCRIPTION
**Pull request by bellex3**
## Summary

1) Added "Local Move" in the Orders type dropdown. Screenshots are below to show where in the Customer and Office apps UI the orders dropdown has changed.
2) Added a migration file to update the comment on orders_type to include "Local Move" and deleted an outdated mention "Permanent Change of assignment (PCA)"
3) Added Tests to test for the presence of Local Move in Orders Type dropdown
     - Orders.test.jsx
     - ServicesCounselingOrders.test.jsx
     - OrdersInfoForm.test.jsx
     - EditOrdersForm.test.jsx
     - OrdersDetailForm.test.jsx
4) Updated Storybook components to include Local Move
     - [EditOrdersForm](https://output.circle-artifacts.com/output/job/3f76194d-cbab-42a2-a8e7-85f3a611dbc1/artifacts/0/storybook/index.html?path=/docs/customer-components-forms-edit-orders-form--docs)
     - [OrdersInfoForm](https://output.circle-artifacts.com/output/job/3f76194d-cbab-42a2-a8e7-85f3a611dbc1/artifacts/0/storybook/index.html?path=/docs/customer-components-forms-orders-info-form--docs)

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

**Customer app**
1. Go to http://milmovelocal:3000/ and click on "Local Sign In"
2. Click on "Create a new Milmove User" button on the top right side.
3. Go through the steps of creating a new user and complete the profile section.
4. Once profile section is completed, click on Step 2 Upload Orders, and click on "Add Orders".
5. Under "Orders type" dropdown, you will see "Local Move". Complete filling out Orders section.
6.  Once orders section is complete, click on "Edit" in the top right corner of section.
7. In the Edit Orders form, you'll see Local Move in the orders type dropdown.

**Office app**
1. Go to http://officelocal:3000/ and click on "Local Sign In"
2. Login as on user "services-counselor-20230522203844-8630f901ada0@example.com" (should be 1st one)
3. Under "Counseling" tab, click on the move with Move Code "Q8Y9C9".
4. Scroll down to "Orders" section and you'll see Local Move in the Order type. 
5. Click on "View and edit orders" button on the top right of Orders section.
8. On the right side panel, you'll see Local Move in the Orders type dropdown.

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/about/supported-browsers) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

## Screenshots
Screenshot 2023-05-22 at 1 14 54 PM
Screenshot 2023-05-22 at 1 16 34 PM
Screenshot 2023-05-22 at 1 16 47 PM
Screenshot 2023-05-22 at 1 17 39 PM
Screenshot 2023-05-22 at 1 17 50 PM

### CODE REVIEW

File: src/components/Office/OrdersDetailForm/OrdersDetailForm.test.jsx [hunk 1]
```diff
@@ -48,7 +48,8 @@ const defaultProps = {
   ordersTypeDetailOptions,
   showOrdersAcknowledgement: true,
   validateTac: jest.fn,
-  ordersType: 'PERMANENT_CHANGE_OF_STATION',
+  // ordersType: 'PERMANENT_CHANGE_OF_STATION',
```
Comment (pearl-truss) [inline src/components/Office/OrdersDetailForm/OrdersDetailForm.test.jsx:51]: Nit: I think the commented out code could be removed

**Review by pearl-truss — APPROVED** [review]
I left a small suggestion but otherwise this checks all of the AC and LGTM

**Review by kctruss — APPROVED** [review]
Reviewed your migration - no issues with the update (see screenshot from my local DB), great catch on editing that column comment to include the new order type!

Screenshot 2023-05-23 at 1 08 08 PM

File: src/components/Office/OrdersDetailForm/OrdersDetailForm.test.jsx (hunk 1 above)
Comment (bellex3) [reply src/components/Office/OrdersDetailForm/OrdersDetailForm.test.jsx]: fixed commented out code

[the remark this answers is not in the document]
Comment (bellex3) [reply src/components/Office/OrdersDetailForm/OrdersDetailForm.test.jsx]: fixed commented out code

**Review by rileyorr — APPROVED** [review]
@bellex3 I wasn't able to find that service counselor role in the ephemeral deploy. However, I went into a move as a different service counselor and updated the orders to Local Move and confirmed that it displays in the Orders section on the Move Details tab. Customer app also LGTM.

**Pull request merged**